### ISSUE TITLE
Upcoming updates to Airtable Attachment URLs that are fetched via API

### ISSUE THREAD

**Issue by DannyMatteson-at** [jbolda/gatsby-source-airtable#422]
Hello, I'm with the Developer Support Engineer team at Airtable and we wanted to share this upcoming notification about [Changes to Airtable attachments](https://support.airtable.com/hc/en-us/articles/4852449595671-Changes-to-Airtable-attachments) in which Airtable will be incorporating expiring links functionality starting November 8th, 2022. I don't have full context on how `gatsby-source-airtable` uses these attachment URLs but more details about how this impacts 3rd party integrations can be found in the linked support article. Please contact support@airtable.com with a reference to this GitHub Issue if there are any technical questions we can help address. Thanks.

**Comment by ZeldOcarina** [comment]
Oh no what a bad news..

**Comment by flogy** [comment]
Without having it tested yet, I think this can already be solved by using `fileNode` mappings (see: https://github.com/jbolda/gatsby-source-airtable#using-markdown-and-attachments). Like that, the attachments are downloaded at build time and made available locally. Is this correct @jbolda ?

**Comment by flogy** [comment]
Could not really get it to work so I finally switched over to [gatsby-source-airtable-next](https://github.com/davidpaulsson/gatsby-source-airtable-next) which supports downloading file attachments too.

**Comment by ZeldOcarina** [comment]
Yeah I think the Gatsby team should develop an official plugin for AirTable, it's such an incredible CRM!

**Comment by jbolda** [comment]
@ZeldOcarina I have inquired multiple times in the past of Gatsby corporation would be interested in taking own ownership in the plugin, but there has never been interest.

I have attempted to find others to maintain this plugin as I am no longer involved with the community, and was doing this in my free time time, unpaid. Unfortunately, everyone has continued to fall off or not had interest in helping.

It still seems with these recent API changes, there is renewed interest in putting effort into this plugin. @davidpaulsson seems like you have an updated fork? Is there interest in maintaining and publishing under this package name? Actually separately having other conversations around pretty possible solutions, and asked in #484 if Airtable would be interested in taking an ownership interest.

**Comment by ZeldOcarina** [comment]
This has not been a breaking change for me and I am still using the plugin successfully so far. It even works with the new API tokens, you've really laid out some robust foundations here, kudos to you.
I've answered on #484 too on possible Gatsby UC involvement in this. I see it's still a small organization, but it's where I've been directed too by the Gatsby team when I asked this plugin to enter the core, too..
Not sure why they are disregarding AirTable imo it's hands down the best CRM, way better than Contentful ..

**Comment by davidpaulsson** [comment]
@jbolda It's not so much a fork but another source plugin. I'm up for maintaining and publishing under this package name, but it's a very breaking change from how this pkg works to switch to my version of sourcing from Airtable.

**Comment by jbolda** [comment]
I have decided to [move this plugin over to the GUC](https://github.com/gatsby-uc/plugins/issues/368). It seems that is the best fit for continued maintenance of this plugin.

@davidpaulsson, the adjustments that you have done seem perfectly reasonable to me. Some of that seems to stem from it's creation in a different era of Gatsby. Either way, this plugin hasn't seen any major versions for quite some time so it may very well make sense to combine efforts. I will leave that discussion to the GUC though.
